usr/local/www/diag_dump_states_sources.php

```diff
@@ -104,54 +102,41 @@ function(index,row) { jQuery(row).fadeOut(1000); }
 //]]>
 </script>
 
-<table width="100%" border="0" cellpadding="0" cellspacing="0">
-	<tr>
-		<td>
-		<?php
-			$tab_array = array();
-			$tab_array[] = array(gettext("States"), false, "diag_dump_states.php");
-			$tab_array[] = array(gettext("Source Tracking"), true, "diag_dump_states_sources.php");
-			$tab_array[] = array(gettext("Reset States"), false, "diag_resetstate.php");
-			display_top_tabs($tab_array);
-		?>
-		</td>
-	</tr>
-	<tr>
-		<td>
-			<div id="mainarea">
+
+<div class="panel panel-default">
+
+<?php
+	$tab_array = array();
+	$tab_array[] = array(gettext("States"), false, "diag_dump_states.php");
+	$tab_array[] = array(gettext("Source Tracking"), true, "diag_dump_states_sources.php");
+	$tab_array[] = array(gettext("Reset States"), false, "diag_resetstate.php");
+	display_top_tabs($tab_array);
+?>
 
 <!-- Start of tab content -->
+	<br />
+	<form action="<?=$_SERVER['SCRIPT_NAME']?>" method="get">
+		<div class="pull-right">
+			<?=gettext("Filter expression:")?>
+			<input type="text" name="filter" class="formfld search" value="<?=htmlspecialchars($_GET['filter'])?>" size="30" />
+			<input type="submit" class="btn btn-primary btn-sm formbtn" value="<?=gettext("Filter")?>" />
+			&nbsp;
+		</div>
+	</form>
+	<br>
+	<div class="responsive">
+	<table class="table table-striped table-hover table-compact">
+		<thead>
+			<tr>
+				<th class="listhdrr" width="40%"><?=gettext("Source -> Destination")?></th>
```

Comment:
Sjon,

This looks like an old version.

Standby while I check please. There is no way I left all that junk in these files.

Steve

> On Apr 28, 2015, at 2:44 PM, SjonHortensius <notifications@github.com> wrote:
> 
> In usr/local/www/diag_dump_states_sources.php <https://github.com/SjonHortensius/pfsense/pull/97#discussion_r29274787>:
> 
> >  
> >  <!-- Start of tab content -->
> > +	<br />
> > +	<form action="<?=$_SERVER['SCRIPT_NAME']?>" method="get">
> > +		<div class="pull-right">
> > +			<?=gettext("Filter expression:")?>
> > +			<input type="text" name="filter" class="formfld search" value="<?=htmlspecialchars($_GET['filter'])?>" size="30" />
> > +			<input type="submit" class="btn btn-primary btn-sm formbtn" value="<?=gettext("Filter")?>" />
> > +			&nbsp;
> > +		</div>
> > +	</form>
> > +	<br>
> > +	<div class="responsive">
> > +	<table class="table table-striped table-hover table-compact">
> > +		<thead>
> > +			<tr>
> > +				<th class="listhdrr" width="40%"><?=gettext("Source -> Destination")?></th>
> all these classes are not required (the widths neither)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SjonHortensius/pfsense/pull/97/files#r29274787>.
> 

Steve
--
Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com <mailto:sbeaver@netgate.com> | www.netgate.com <http://www.netgate.com/>